### PR TITLE
Christines htmlexercise 1 branch

### PR DESCRIPTION
**Pull request by chriscyber**
I intended to remove the first commit - not sure if that worked.

### CODE REVIEW

File: christine-anthony.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+<html>
+
+  <head>
+    <meta charset="uft-8">
+  <!-- why doesnt the fond change??? -->
```
Comment (tkraak) [inline christine-anthony.html]: Meta charset does not have anything to do with fonts. See [this intro](https://www.w3.org/International/questions/qa-what-is-encoding).

File: christine-anthony.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <title>Of Patterns and Power: Web Standards Then & Now </title>
+      <meta name="description" content="HTML article titled Of Patterns and Power: Web Standards Then and Now with many links to other articles.">
+
+
+  </head>
+
+  <body>
+    <header>
+      <img source="images/exe1-1.jpeg" alt="HTML5 Doctor Logo"WIDTH=32 HEIGHT=32>
+    <!-- why doesnt it know where to get the img??? -->
```
Comment (tkraak) [inline christine-anthony.html]: Do  you have an image by that file name in your `images` directory?

File: christine-anthony.html (hunk 1 above)
Comment (chriscyber) [reply christine-anthony.html]: I understand charsets now. I really like the last article by Joel Spolsky linked at the end of the intro about unicode and character sets.

File: christine-anthony.html (hunk 2 above)
Comment (chriscyber) [reply christine-anthony.html]: its in the images directory in the html exercise-1 that i forked/cloned for this project. wouldn't that work? I'll try out a few other things...

**Pull request merged**